pkg/gofr/http/errors.go

```diff
@@ -94,3 +119,26 @@ func (ErrorPanicRecovery) Error() string {
 func (ErrorPanicRecovery) StatusCode() int {
 	return http.StatusInternalServerError
 }
+
+func (ErrorPanicRecovery) LogLevel() logging.Level {
+	return logging.ERROR
+}
+
+// validate the errors satisfy the underlying interfaces they depend on.
+var (
+	_ statusCodeResponder = ErrorEntityNotFound{}
+	_ statusCodeResponder = ErrorEntityAlreadyExist{}
+	_ statusCodeResponder = ErrorInvalidParam{}
+	_ statusCodeResponder = ErrorMissingParam{}
+	_ statusCodeResponder = ErrorInvalidRoute{}
+	_ statusCodeResponder = ErrorRequestTimeout{}
+	_ statusCodeResponder = ErrorPanicRecovery{}
+
+	_ logging.LogLevelResponder = ErrorEntityNotFound{}
+	_ logging.LogLevelResponder = ErrorEntityAlreadyExist{}
+	_ logging.LogLevelResponder = ErrorInvalidParam{}
+	_ logging.LogLevelResponder = ErrorMissingParam{}
+	_ logging.LogLevelResponder = ErrorInvalidRoute{}
+	_ logging.LogLevelResponder = ErrorRequestTimeout{}
+	_ logging.LogLevelResponder = ErrorPanicRecovery{}
+)
```

Comment:
wait I don't get why you get so many changes in a PR about S3.

Is it a broken rebase? Or the bad target branch?